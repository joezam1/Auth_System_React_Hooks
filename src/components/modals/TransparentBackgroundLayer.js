import React from 'react';


//Test: DONE
export default function TransparentBackgroundLayer(){

    return(<div className="transparent-layer-section transparency background-layer"></div>);
}