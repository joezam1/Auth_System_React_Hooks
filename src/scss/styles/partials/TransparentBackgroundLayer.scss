.box {
    display: flex;
    flex-flow: column;
    height: 100%;
}

.transparency{
    background-color: rgba(0,0,0,.5);
    backdrop-filter: blur(5px);
    color: #fff;

}

.background-layer{
    border:0px solid red;
    height: 100vh;
    position:absolute;
    width: 100%;
    z-index: 200;
}