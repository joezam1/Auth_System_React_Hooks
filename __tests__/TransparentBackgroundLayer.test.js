import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import { render, cleanup } from '@testing-library/react';

import TransparentBackgroundLayer from "../src/components/modals/TransparentBackgroundLayer.js";


describe('File: TransparentBackgroundLayer.js', function(){
    afterEach(cleanup);
    afterAll(()=>{jest.resetAllMocks(); });

    describe('Component: TransparentBackgroundLayer', function(){

        test('It can take a snapshot', function(){
            let container1 = render( <MemoryRouter> <TransparentBackgroundLayer /> </MemoryRouter>);
            expect(container1.asFragment(<TransparentBackgroundLayer/> )).toMatchSnapshot();
        });
    });
});
