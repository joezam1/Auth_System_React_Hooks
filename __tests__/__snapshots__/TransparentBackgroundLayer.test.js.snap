// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`File: TransparentBackgroundLayer.js Component: TransparentBackgroundLayer It can take a snapshot 1`] = `
<DocumentFragment>
   
  <div
    class="transparent-layer-section transparency background-layer"
  />
   
</DocumentFragment>
`;
